audits/manifest/background-color.js

```diff
@@ -26,13 +26,16 @@ class ManifestBackgroundColor {
     return 'Contains background_color';
   }
 
-  static audit(inputs) {
-    let hasBackgroundColor = false;
-    const manifest = inputs.manifest;
+  static hasBackgroundColorValue(manifest) {
+    /* eslint-disable camelcase */
+    return Boolean(manifest !== undefined &&
+                   manifest.background_color !== undefined &&
```

Comment:
Depends how safe we want to be, given that there is no formal enforcement of how this will be called I would rather be very verbose as compared to the code bailing out here